<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EbDialogSelectUser</class>
 <widget class="QDialog" name="EbDialogSelectUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonMyGroup">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>my group</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMyContact">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>my contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMyEnterprise">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>my enterprise</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelected">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>selected</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSelected">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>231</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>150</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
